scss/detail.scss

```diff
@@ -56,3 +57,22 @@
 .book-separator {
   border-bottom: 4px solid $earls-green;
 }
+
+.book-image-container {
+  &::after {
+  -ms-transform: rotate(15deg);
+  -webkit-transform: rotate(15deg);
+  background: url('../assets/badge.png');
+  background-repeat: no-repeat;
+  background-position: 1px;
+  background-size: 100%;
+  bottom: 830px;
```

Comment:
It's not a good idea to use such big number for this. You shoul be able to predict it's behavior by just reading it. You can use top instead or 

`bottom: calc(100% + some small value);`